test/Microsoft.ML.Functional.Tests/Prediction.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.SamplesUtils;
+using Xunit;
+
+namespace Microsoft.ML.Functional.Tests
+{
+    public partial class PredictionScenarios
+    {
+        /// <summary>
+        /// Reconfigurable predictions: The following should be possible: A user trains a binary classifier,
+        /// and through the test evaluator gets a PR curve, the based on the PR curve picks a new threshold
+        /// and configures the scorer (or more precisely instantiates a new scorer over the same predictor)
+        /// with some threshold derived from that.
+        /// </summary>
+        [Fact(Skip = "Blocked by issue #2465")]
```

Comment:
The test can still be run, right?  Since you are commenting out the code that is being blocked.  Maybe remove the Skip here, and add a TODO to the commented out line below.